src/components/GameListPage.jsx

```diff
@@ -17,14 +17,14 @@ class GameListPage extends Component {
     return (
       <div className="gameListPage container">
         <ListGroup>
-          {this.props.roomIds.map(roomId => {
+          {this.props.rooms.map(room => {
             return (
               <ListGroupItem
-                key={roomId}
+                key={room.roomId}
                 tag="button"
                 onClick={(event) => this.props.joinRoom(roomId)}
                 action>
-                {roomId}
+                {room.roomId}
                 <Badge pill className="float-right">0</Badge>
```

Comment:
@duyoji 
Is this part for the number of players ?

Reply:
@ryutaudo 

Yes.
According to your implementation of rooms, Maybe you can write code like below.

```js
<Badge pill className="float-right">{this.props.room.numberOfPlayers}</Badge>
```